### PR TITLE
Old router

### PR DESCRIPTION
**Pull request by cemersoz**
fixed issue https://github.com/learnlinux/tuxlab-app/issues/26
added mongo connection to api

### CODE REVIEW

File: client/imports/ui/pages/explore/explore.html [hunk 1]
```diff
@@ -11,8 +11,6 @@
 					<form (ngSubmit)="findCourse()">
 						<input placeholder="Find Course" name="cname" type="text" id="search-input">
 						<input type="submit" style="display: none">
-						<input placeholder="Find Course" name="cname" type="text" id="search-input" />
```
Comment (DerekTBrown) [inline client/imports/ui/pages/explore/explore.html]: Let's not delete Sander's code

File: client/tuxlab.html [hunk 2]
```diff
@@ -89,48 +89,3 @@
     </md-content>
 </md-sidenav-layout>
 
-          <!-- Navigation -->
```
Comment (DerekTBrown) [inline client/tuxlab.html]: Let's not delete this either

File: tests/gagarin/tuxlab.api.js [hunk 3]
```diff
@@ -2,7 +2,7 @@
     using Meteor Gagarian (https://github.com/anticoders/gagarin)
 **/
 describe('tuxlab.task tests',function(){
-  var tuxlab = require('../../server/api/tuxlab.js');
```
Comment (DerekTBrown) [inline tests/gagarin/tuxlab.api.js]: Please don't use relative paths.  Just declare the TuxLab package in the overall package.json, and then reference it by package name

File: client/imports/ui/pages/explore/explore.html (hunk 1 above)
Comment (cemersoz) [inline client/imports/ui/pages/explore/explore.html]: you're wrong

File: server/imports/api/labMongo.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+var labMongo = function(){}
+labMongo.prototype.mongo = mongo;
+labMongo.prototype.mongoClient = mongoClient;
+
+labMongo.prototype.importLab = function(courseId,labId,callback){
+  var slf = this;
+  mongoClient.connect(url, function(err,db){
+    if(err){
+      console.log("you fucked up");
```
Comment (DerekTBrown) [inline server/imports/api/labMongo.js]: This is unacceptable

**Pull request merged**